src/components/TabBar/TabBar.css

```diff
@@ -41,11 +41,4 @@
     border-color: var(--primaryLinkColor);
     color: color(var(--colorMuted) lightness(85%));
   }
-
-  @media only screen and (max-width: 1439px) {
-    white-space: nowrap;
```

Comment:
Let me check it out on your cloud box, maybe it's not as bad as I thought. What's the URL again?